eng/pipelines/common/xplat-setup.yml

```diff
@@ -175,7 +175,7 @@ jobs:
         # Official build OSX pool
         ${{ if and(in(parameters.osGroup, 'osx', 'maccatalyst', 'ios', 'iossimulator', 'tvos', 'tvossimulator'), ne(variables['System.TeamProject'], 'public')) }}:
           name: "Azure Pipelines"
-          vmImage: 'macos-13-arm64'
+          vmImage: 'macos-latest-internal'
```

Comment:
These have limited disk. Are we OK with this?

Reply:
They actually have enough space and we've been silently running on this pool for the last week. It was just the testing image that had limited space.